### DISCUSSION TITLE
Naming: "EMF" or "Ecore"?

### DISCUSSION THREAD

**Discussion by alessiostalla** [Strumenta/StarLasu#26] [Polls]
We don't have a well-defined naming strategy across StarLasu. For example, we have [EMFEnabledParser](https://javadoc.io/doc/com.strumenta.kolasu/kolasu-emf/latest/com/strumenta/kolasu/emf/EMFEnabledParser.html) in Kolasu, but EcoreEnabledParser in Tylasu.

@ftomassetti @loradd

**Comment by loradd** [comment]
I would use Ecore 👍

**Comment by ftomassetti** [comment]
I think Lorenzo is the one with more experience on EMF/Ecore so I would second his vote

**Comment by alessiostalla** [comment]
I would have gone with Ecore too but didn't want to influence the vote :)
